docker/haal_centraal_brp/data/test-data-changelog.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This is the way I thought about it but I am wondering if there is a more straight forward/simpler way to keep track of these changes, since we cannot add comments to the json file.